### PR TITLE
Setting page

### PR DESCRIPTION
**Pull request by EJEunjiyaz**
I completely build the setting page with css, add `views` and `urls` in django. Please final check for setting.html and setting.css file.

### CODE REVIEW

**Comment by kornkritpawit** [comment]
Overall is good.

**Comment by Klan300** [comment]
- [x]  can you fix setting to a block on div '.box' 
- [x]  add link font that I require to setting.html 
`<link href="https://fonts.googleapis.com/css?family=Varela+Round&display=swap" rel="stylesheet">`
- [x] add css font to setting.css
`* {
font-family: 'Varela Round', sans-serif;
}`

- [x] don't use `<br>` pls use padding or margin to create space

- [x] pls put `<script>` below `<body>`

- [x] pls put `<meta>` `<link>` `<title>` into `<head>` tags

> it have another way to create div box by not using table just kind link `display : flex`  on css
> please read and try this to your code "[flexbox](https://internetingishard.com/html-and-css/flexbox/)"

ขอโทษที่ให้แก้เยอะ แต่เพื่อความเป็นระเบียบของงานแล้วก็ เวลาแต่งจะได้ง่ายแล้วก็สวยมากขึ้น สียังไม่ลงตัวค่อยเพิ่มได้ ถ้ามึงทำได้ มึงจะทำได้อีกเยอะ

**Comment by Parkkung** [comment]
not necessary to use `<br>`

**Comment by EJEunjiyaz** [comment]
I already add flexbox and padding. Please check again.
Thank you

**Comment by Klan300** [comment]
Screen Shot 2562-11-16 at 15 03 18
I think It too much big more than it should be. it should have space between time and sound on-off.

**Review by Klan300 — CHANGES_REQUESTED** [review]
please fix some design on your settings page.

**Review by EJEunjiyaz — COMMENTED** [review]
Finish

**Comment by EJEunjiyaz** [comment]
image

**Comment by EJEunjiyaz** [comment]
Please check again @Klan300 . Thank you for always good reviews.

**Review by Klan300 — APPROVED** [review]

**Review by Parkkung — APPROVED** [review]

File: Cardgame/static/css/setting.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    right: 0;
+    bottom: 0;
+    background-color: #ccc;
+    -webkit-transition: .4s;
+    transition: .4s;
+}
+  
+.slider:before {
+    position: absolute;
+    content: "";
```
Comment (EJEunjiyaz) [inline Cardgame/static/css/setting.css:91]: Remove This

**Review by EJEunjiyaz — COMMENTED** [review]
Should fix these format first

**Pull request merged**